pkg/cli/debug/debug.go

```diff
@@ -486,6 +487,10 @@ func (o *DebugOptions) RunDebug() error {
 	o.Annotations[debugPodAnnotationSourceResource] = fmt.Sprintf("%s/%s", infos[0].Mapping.Resource, infos[0].Name)
 	o.Annotations[debugPodAnnotationSourceContainer] = o.ContainerName
 
+	if infos[0].Mapping.GroupVersionKind.Group == "Node" {
```

Comment:
This is not equal to the parent patch https://github.com/openshift/oc/pull/1763/files#diff-632e7c700936a9f52eab0117ec7594600dfc6154845643796c5e277a7c17abf3R490?